integration/tests/inferentia/inferentia_test.go

```diff
@@ -57,6 +59,10 @@ var _ = BeforeSuite(func() {
 	clusterWithoutPlugin = noInstallCluster
 	clusterWithNeuronPlugin = defaultCluster
 
+	nodeType := []string{"inf1.xlarge", "inf2.xlarge"}
+	randomIndex := rand.Intn(len(nodeType))
+	selectedNodeType = nodeType[randomIndex]
```

Comment:
With this random selection of the instance type, it's possible to get many consecutive runs with the same instance type. To have a more consistent spread of the instance types, we can, for example, use the current date to alternate between the two instance types.